examples/with-changesets/.github/workflows/release.yml

```diff
@@ -21,14 +21,14 @@ jobs:
           node-version: 16.x
 
       - name: Install Dependencies
-        run: yarn
+        run: pnpm i
 
       - name: Create Release Pull Request or Publish to npm
         id: changesets
         uses: changesets/action@v1
         with:
           # This expects you to have a script called release which does a build for your packages and calls changeset publish
-          publish: yarn release
+          publish: pnpm publish
```

Comment:
This should still reference the `release` command in the package.json

Reply:
🤦